node-hub/dora-llama-cpp-python/dora_llama_cpp_python/main.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pyarrow as pa
+from dora import Node
+from pathlib import Path
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+# Environment variables for model configuration
+SYSTEM_PROMPT = os.getenv(
+    "SYSTEM_PROMPT",
+    "You're a very succinct AI assistant with short answers.",
+)
+MODEL_LOCAL_PATH = os.getenv("MODEL_LOCAL_PATH", "")  # Local model path takes precedence
+MODEL_NAME = os.getenv("MODEL_NAME", "TheBloke/Llama-2-7B-Chat-GGUF")  # HF repo as fallback
```

Comment:
Sorry for the additional change but can we name this, mod
```suggestion
MODEL_NAME_OR_PATH = os.getenv("MODEL_NAME_OR_PATH", "TheBloke/Llama-2-7B-Chat-GGUF") 
```

This way it Is easier to use one or the other. This name comes from transformer naming within `from_pretrained`

Reply:
ohk 👍️